command/server/tls_util.go

```diff
@@ -61,7 +61,7 @@ func GenerateCert(caCertTemplate *x509.Certificate, caSigner crypto.Signer) (str
 		NotAfter:       time.Now().Add(365 * 24 * time.Hour),
 		NotBefore:      time.Now().Add(-1 * time.Minute),
 		IPAddresses:    []net.IP{net.ParseIP("127.0.0.1")},
-		DNSNames:       []string{"localhost", "localhost4", "localhost6", "localhost.localdomain"},
+		DNSNames:       []string{hostname, "localhost", "localhost4", "localhost6", "localhost.localdomain"},
```

Comment:
What would occur here if the `hostname` var was `localhost`, would two entries with the same value cause any issues in the generated cert?

Reply:
Yeah, I don't think that'd cause any problems, but I can fix.